requirements.txt

```
@@ -1,60 +1,59 @@
-Flask==0.12.4
-Werkzeug==0.11.11
-Jinja2==2.8
-itsdangerous==0.24
-click==6.6
-MarkupSafe==0.23
-pyOpenSSL==17.5.0
-httplib2==0.10.3
+Flask==1.1.1
+Jinja2==2.10.3
+itsdangerous==1.1.0
+click==6.7
+MarkupSafe==1.1.1
+pyOpenSSL==19.0.0
+httplib2==0.14.0
 wtforms==2.2.1
-Flask-RESTful==0.3.5
-Flask-Login==0.4.0
+Flask-RESTful==0.3.7
+Flask-Login==0.4.1
 Flask-OAuthLib==0.9.5
 # pin this until https://github.com/lepture/flask-oauthlib/pull/388 is released
 requests-oauthlib>=0.6.2,<1.2.0
-Flask-SQLAlchemy==2.3.2
-Flask-Migrate==2.0.1
+Flask-SQLAlchemy==2.4.1
+Flask-Migrate==2.5.2
 flask-mail==0.9.1
-flask-talisman==0.6.0
+flask-talisman==0.7.0
 Flask-Limiter==0.9.3
-passlib==1.6.2
-aniso8601==1.1.0
-blinker==1.3
-psycopg2==2.7.3.2
+passlib==1.7.1
+aniso8601==8.0.0
+blinker==1.4
+psycopg2==2.8.3
 python-dateutil==2.8.0
-pytz==2016.7
-PyYAML==3.12
-redis==3.2.1
-requests==2.21.0
+pytz==2019.3
```

```suggestion
pytz>=2019.3
```

Probably safe to keep upgrading automatically.